app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/previous_return_exists_with_same_completion_date.scala.html

```diff
@@ -20,44 +20,87 @@
 
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
-@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.UserType
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.TimeUtils
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.ReturnSummary
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.address.Address
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers
 
 @this(
         mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
         pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
         backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
 )
 
-@(isATrust: Boolean, backLink: Call)(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
+@(previousReturn: Option[ReturnSummary], backLink: Call)(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
 
 @key = @{"previousReturnExistsWithSameCompletionDate"}
-@userKey = @{
-    if(request.userType.contains(UserType.Agent)) ".agent"
-    else if(isATrust) ".trust"
-    else ""
-}
+@title = @{ messages(s"$key.title") }
+@submissionId = @{
+    previousReturn match {
+        case Some(value) => value.submissionId
+        case None => ""
+    }
 
+}
 
-@title = @{messages(s"$key.title")}
+@line1Display = @{
+    previousReturn match {
+        case Some(value) => value.propertyAddress match {
+            case Address.UkAddress(line1, _, _, _, _) => line1
+            case Address.NonUkAddress(_, _, _, _, _, _) => ""
+        }
+        case None => ""
+    }
+  }
+
+@line2Display = @{
+    previousReturn match {
+        case Some(value) => value.propertyAddress match {
+            case Address.UkAddress(_, _, _, Some(town), postcode) => s"$town, ${postcode.value}"
```

Comment:
do we not want to show the full address?

Reply:
Noo. The mockups looked like.. 
![image](https://user-images.githubusercontent.com/9150276/93067577-4ce57700-f673-11ea-8220-a626544887d7.png)

Reply:
that might just be the way the data was set up with some of the address lines blank. It'd be good to double check if all lines of address should be displayed.